pyhf/modifiers/histosys.py

```diff
@@ -46,7 +46,18 @@ def pdf(self, a, alpha):
 
     def apply(self, channel, sample, pars):
         assert int(pars.shape[0]) == 1
-        return interpolator(0)(self.at_minus_one[channel['name']][sample['name']],
-                               self.at_zero[channel['name']][sample['name']],
-                               self.at_plus_one[channel['name']][sample['name']],
-                               pars)[0]
+
+        tensorlib, _ = get_backend()
+        results = interpolator(0)(
+            tensorlib.astensor([
+                [
+                    [
+                        self.at_minus_one[channel['name']][sample['name']],
+                        self.at_zero[channel['name']][sample['name']],
+                        self.at_plus_one[channel['name']][sample['name']]
+                    ]
+                ]
+            ]), tensorlib.astensor([tensorlib.tolist(pars)])
```

Comment:
why to we call `tolist` here?

Reply:
This gets factored out soon enough. It's a bandage for this inconsistency:

```
pars = [1.0]
np.array([pars]) == [[1.0]]
tf.Tensor([pars]) == [[1.0]]
torch([pars]) == [1.0]
```